ccdproc/image_collection.py

```diff
@@ -216,6 +222,13 @@ def files(self):
         """
         return self._files
 
+    @property
+    def ext(self):
+        """
+        str or int, The extension from which the header and data will be read in all files.
```

Comment:
I suspect that the `tuple` works because it works for [`fits.getheader`](http://docs.astropy.org/en/stable/io/fits/api/files.html#astropy.io.fits.getdata). But I might be wrong.